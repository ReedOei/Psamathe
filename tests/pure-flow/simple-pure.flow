;

2 --> var n : nat

