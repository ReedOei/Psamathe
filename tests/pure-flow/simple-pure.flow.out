<generatedTop>
  <common>
    <typeEnv>
      .Map
    </typeEnv>
    <types>
      .Map
    </types>
    <functions>
      .Map
    </functions>
  </common>
  <k>
    .
  </k>
  <lookup>
    n |-> selected ( loc ( 0 ) , loc ( 0 ) )
  </lookup>
  <storage>
    loc ( 0 ) |-> resource ( nat , ListItem ( 2 ) )
    loc ( 1 ) |-> resource ( nat , ListItem ( 0 ) )
  </storage>
  <catchStack>
    .List
  </catchStack>
  <createdVals>
    .Map
  </createdVals>
</generatedTop>
